reword execution quota disabled > not set (#1124)
MicroEJ/docs: KernelDeveloperGuide/kf.rst

suspended (the Feature is paused).

> If a Feature has reached its execution quota and is holding a monitor (through one of its threads), 

Reviewer: ```suggestion
If a Feature exceeds its execution quota while holding a monitor (via one of its threads),
```

> and another Module (Feature or Kernel) with no execution quota set attempts to acquire the same monitor (through one of its threads), 

Reviewer: ```suggestion
and another Module (Feature or Kernel) with no execution quota limit attempts to acquire the same monitor (via one of its threads), 
```
